src/components/CompareResults/ResultsMain.tsx

```diff
@@ -20,13 +27,46 @@ function ResultsMain() {
     title: style({
       margin: 0,
       marginBottom: Spacing.Medium,
+      fontStyle: 'Heading XS',
+      fontFamily: 'Metropolis',
+      fontSize: '24px',
+      fontWeight: 700,
+      lineHeight: '28px',
+      letterSpacing: '-0.01em',
+      textAlign: 'left',
     }),
+    subtitle: style({
+      borderLeft: '1px solid #5B5B66',
+      paddingLeft: '9px',
+      fontFamily: 'SF Pro',
+      fontSize: '15px',
+      fontWeight: 400,
+      lineHeight: '20px',
+      height: '20px',
+      textAlign: 'left',
+    }),
+    titleContainer: style({
+      alignItems: 'baseline',
+      gap: '9px',
+    }),
+  };
+
+  const subtitles = {
+    'compare-results': 'Compare with a base',
+    'compare-over-time-results': 'Compare over time',
   };
 
   return (
     <Container className={styles.container} data-testid='results-main'>
       <header>
-        <div className={styles.title}>Results</div>
+        <Grid container className={styles.titleContainer}>
+          <Grid item className={styles.title}>
+            Results
+          </Grid>
+          <Grid item className={styles.subtitle}>
+            {subtitles[view]}
```

Comment:
How about enclosing the title and subtitle in `h `tags?

Reply:
What h tags were you thinking? Can you give me an example?